gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -2417,6 +2466,13 @@ gnc_ppr_update_status_query (GncPluginPageRegister *page, gboolean refresh)
     gnc_plugin_page_register_set_filter_tooltip (page);
 
     if (refresh)
+    // clear previous filter query
```

Comment:
Oops. ```if (refresh)``` needs to be *after* this block. That confused me when I looked at the next commit.

Reply:
This was a mistake in trying to separate work done into the two commits.